xarray/core/variable.py

```diff
@@ -633,11 +633,18 @@ def to_index(self) -> pd.Index:
         """Convert this variable to a pandas.Index"""
         return self.to_index_variable().to_index()
 
-    def to_dict(self, data: bool = True, encoding: bool = False) -> dict:
+    def to_dict(
+        self, data: bool = True, encoding: bool = False, numpy_data: bool = False
```

Comment:
only minor comment is whether `data_as_numpy` is clearer? Since `.as_numpy` is what we'll want to use